rpcserver.go

```diff
@@ -1409,15 +1423,15 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 		sweepTXID := sweepTxPkg.SweepTx.TxHash()
 		txid = &sweepTXID
 	} else {
-
-		// We'll now construct out payment map, and use the wallet's
+		// We'll now construct out payment map, and use the lnWallet's
 		// coin selection synchronization method to ensure that no coin
 		// selection (funding, sweep alls, other sends) can proceed
-		// while we instruct the wallet to send this transaction.
+		// while we instruct the lnWallet to send this transaction.
 		paymentMap := map[string]int64{targetAddr.String(): in.Amount}
```

Comment:
the `in.Amount` here needs to point to the sum of values of all selected utxos

Reply:
why would that be?